src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/KnownCosmosDbCmkSubStatusValue.cs

```diff
@@ -11,15 +11,14 @@ namespace Microsoft.Health.Fhir.CosmosDb.Features.Storage
     public enum KnownCosmosDbCmkSubStatusValue
```

Comment:
FYI: I broke out this enum into "server-side" and "client side" issues based on the categorizations listed in https://docs.microsoft.com/en-us/rest/api/cosmos-db/http-status-codes-for-cosmosdb#http-substatus-codes